lib/devices/gen3/shelly0110dimg3.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const shellyHelperGen2 = require('../gen2-helper');
+
+/**
+ * Shelly Plus 0-10V Dimmer Gen 3 / shelly0110dimg3
+ *
+ * https://shelly-api-docs.shelly.cloud/gen2/Devices/Gen2/ShellyPlus10V
```

Comment:
Should be https://shelly-api-docs.shelly.cloud/gen2/Devices/Gen3/ShellyDimmer0110VPMG3

Reply:
corrected Link